Src/StartMenu/StartMenu.rc

```diff
@@ -54,10 +54,10 @@ END
 
 // Solution loading fail-safe
 #ifndef _PRODUCT_VERSION
-#define _PRODUCT_VERSION 4.4.102
```

Comment:
These version related changes are not needed.
Version is automatically changed during build process.

There is no point to update it in sources (as it changes with every official build).